Tests/SwiftDocCTests/Semantics/ChoiceTests.swift

```diff
@@ -233,7 +233,7 @@ Choice @1:1-9:2 isCorrect: true
                 let expectedDump = """
 Choice @1:1-7:2 isCorrect: true
 ├─ MarkupContainer (empty)
-├─ ImageMedia @2:4-2:11 source: 'ResourceReference(bundleIdentifier: "org.swift.docc.example", path: "blah.png")' altText: 'blah'
+├─ ImageMedia @2:4-2:39 source: 'ResourceReference(bundleIdentifier: "org.swift.docc.example", path: "blah.png")' altText: 'blah'
```

Comment:
Hmm, why did these column numbers change?

Reply:
See more info here apple/swift-markdown#49.

After apple/swift-markdown#50, this column needs to be changed. Maybe we can discuss this change on apple/swift-markdown#50 for whether this change should be made.

Reply:
Ah I see. Yes I think we should open a separate PR for this column changes, if we take the swift-markdown change.

Reply:
So on this PR, we use the original buggy workaround and then a PR both on swift-markdown and swift-docc to fix this?

Reply:
But this PR needs the feature of apple/swift-markdown#50, we certainly can decouple them. But we'll use a buggy implementation as a temporary workaround. See the full context on apple/swift-markdown#49 of other.

Reply:
I think it would be beneficial to decouple them so that we can merge the swift-markdown PR independently from this one. Otherwise we won't be able to merge the Swift-Markdown changes without this PR